File: Desenvolupament d'interfícies/App_Escritorio/Usuario.cpp
#include "usuario.h"

#include <QDebug>
#include <math.h>
#include <stdlib.h>

Usuario::Usuario(int id, QString nombre, QString apellidos, QString correo, QString contrasena, double latitud, double longitud): id(id), nombre(nombre), apellidos(apellidos), correo(correo), contrasena(contrasena), latitud(latitud), longitud(longitud) {
}

Usuario::Usuario() {
}

File: Programació multimèdia i dispositius mòbils/app/src/main/java/com/example/app_grupob/pojos/Usuario.kt
package com.example.app_grupob.pojos

import androidx.room.Entity
import androidx.room.PrimaryKey

data class Usuario(
    val id: Int,
    val nombre:String,
    val apellidos:String,
    val correo:String,
    val contrasena:String,
    val longitud:Double,
    val latitud:Double
)

@Entity(tableName = "usuarios")
data class UsuarioEntity(
    @PrimaryKey(autoGenerate = true) val id : Int?,
    val nombre:String,
    val apellidos:String,
    val correo:String,
    val contrasena:String,
    val longitud:Double,
    val latitud:Double
)